dashboard/app/assets/javascripts/client_state.js

```diff
@@ -136,5 +136,29 @@ dashboard.clientState.recordVideoSeen = function (videoId) {
   }
 };
 
+dashboard.clientState.hasSeenCallout = function(calloutId) {
+  var calloutsSeenJson = localStorage.getItem('calloutsSeen') || '{}';
+  try {
+    var calloutsSeen = JSON.parse(calloutsSeenJson);
+    return calloutsSeen[calloutId] == 1;
```

Comment:
Why 1 vs. true?  Seeing as this is all client side I am sure the difference would be extremely hard to measure.